emission/integrationTests/docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: "3"
+services:
+  web-server:
+    build: 
+      context: .
+    depends_on:
+      - db
+      - nominatim
+    environment:
+      - DB_HOST=db
+      - WEB_SERVER_HOST=0.0.0.0
```

Comment:
do we need to export the webserver here?

Reply:
Nope! Test passed without it.